migrations/20161013143349-projects.js

```diff
@@ -0,0 +1,36 @@
+exports.up = function (knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('projects', function (table) {
+      table.increments('id').primary()
+      table.string('title')
+      table.string('description')
+      table.string('repo_url')
+      table.string('app_url')
+      table.date('date')
+    }),
+    knex.schema.createTable('photos', function (table) {
+      table.increments('id').primary()
+      table.string('url')
+      table.string('caption')
+      table.integer('projects_id').references('projects.id')
```

Comment:
Sorry I didn't catch this during my code review, but this needs to be renamed to `project_id`.